frontend/src/components/gotEnzymes/Landing.vue

```diff
@@ -99,24 +97,28 @@
               <hr class="mt-6" />
               <h4 id="how-it-was-made" class="is-info is-size-4">How it was made</h4>
               <p>
-                Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a
-                piece of classical Latin literature from 45 BC, making it over 2000 years old.
-                Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked
-                up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and
-                going through the cites of the word in classical literature, discovered the
-                undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de
-                Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45
-                BC. This book is a treatise on the theory of ethics, very popular during the
-                Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes
-                from a line in section 1.10.32.
+                Data in this database are predicted from available advanced deep learning tools:
+                k<sub>cat</sub> published in <i>Nature Catalysis</i> in 2022 (<a
+                  href="https://doi.org/10.1038/s41929-022-00798-z"
+                  >doi:10.1038/s41929-022-00798-z</a
+                >). Our vision for this database is to store every possible predicted parameter for
+                any available enzyme to facilitate computational applications such as
+                flux simulations, and for use in metabolic engineering.
```

Comment:
```suggestion
                flux simulations and metabolic engineering.
```

Reply:
The above suggestion can generate more confusion, since metabolic engineering is not a type of computational application; perhaps it would be clearer to add an `also` after `and`.

Reply:
Then probably separate them in two sentences.

Reply:
Is this what you mean @nanjiangshu ?
```
Our vision for GotEnzymes is to facilitate computational applications, such as
flux simulations, and to improve the Design-Build-Test-Learn cycle in metabolic engineering,
by suggesting candidate alternative enzymes. To this end, we aim to store every
possible predicted parameter for all available enzyme in a single database that is
equally easy to use manually and programatically.
```

Also pinging @feiranl and @Yu-sysbio .

Reply:
Looks much better now.

Reply:
The above suggestion was applied in [ebcff15](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1045/commits/ebcff150ae5c301f9d16157ee98aba7d6c0a80cd).